<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formMergeSlideCalibrations</class>
 <widget class="QDialog" name="formMergeSlideCalibrations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge Calibration</string>
  </property>
  <widget class="QPushButton" name="pbHorizontal">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHorPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>631</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHor_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVertPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>631</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbVertical">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMergeCalibration">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Merge Calibration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCameraID">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>102</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHor_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>107</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
